### PR TITLE
Cluster serving saved model support

### PR DESCRIPTION
**Pull request by cyita**
Support Tensorflow saved model in cluster serving

### CODE REVIEW

File: scripts/cluster-serving/config.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@
 model:
   # model path must be set
   path: model
+  # the inputs of the tensorflow model, separated by ","
+  inputs:
```
Comment (Litchilitchy) [inline scripts/cluster-serving/config.yaml]: does this `inputs` the same thing as `shape` in data field?

**Comment by Litchilitchy** [comment]
)

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala [hunk 2]
```diff
@@ -277,22 +277,23 @@ class ClusterServingHelper {
     val parallelNum = if (blasFlag) coreNum else 1
     val model = new InferenceModel(parallelNum)
 
-    modelInputs = modelInputs.filterNot((x: Char) => x.isWhitespace)
```
Comment (Litchilitchy) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala]: just `.trim` is ok

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala [hunk 3]
```diff
@@ -224,26 +224,40 @@ object ClusterServing {
...
+                })
               } else {
-                val res = localModel.doPredict(x)
-                if (res.isTensor) {
-                  res.toTensor
+                // result is table
+                val separator = ","
+                val res = if (modelType == "openvino") {
+                  // TODO: openvino Table support
+                  result.toTable
```
Comment (Litchilitchy) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala]: your if - else clause are totally same?..

File: scripts/cluster-serving/config.yaml (hunk 1 above)
Comment (cyita) [reply scripts/cluster-serving/config.yaml]: inputs refer to the inputs name of the TensorFlow model.

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala [hunk 4]
```diff
@@ -66,6 +66,9 @@ class ClusterServingHelper {
 
   var sc: SparkContext = null
 
+  var modelInputs: String = null
+  var modelOutputs:String = null
```
Comment (Litchilitchy) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala]: style

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala (hunk 2 above)
Comment (cyita) [reply zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala]: all whitespace should be removed.

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala (hunk 3 above)
Comment (cyita) [reply zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala]: fixed

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala (hunk 4 above)
Comment (cyita) [reply zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala]: fixed

**Pull request merged**